Makefile

```diff
@@ -152,6 +155,12 @@ collect-charts: collect-1.0-charts collect-1.1-charts
 .PHONY: collect-templates
 collect-templates: collect-1.0-templates collect-1.1-templates
 
+.PHONY: collect-olm-manifests
+collect-olm-manifests:
+	rm -rf  ${OLM_MANIFEST_OUT_DIR}
```

Comment:
This should be part of `make clean`

Reply:
Make clean already runs `rm -rf "${OUT_DIR}"` so I think it's fine just having it here. If you don't have it here and you run both maistra and servicemesh builds then you end up with manifest directories containing data for both build types so I think it makes sense to leave this here.